pages/Component_Analysis.md

```diff
@@ -282,6 +282,7 @@ and legal teams an opportunity to create solutions for healthy open source usage
 
 | Name | Owner | Licence | Platforms |
 | ---- | ----- | ------- |---------- |
+| [Vulert](https://vulert.com) | Vulert | Commercial / Freemium | Cross Platform / SaaS |
```

Comment:
Could you do this as link reference like all the others? (Look down around line 333)

Reply:
done, sorry for error.